content/docs/guides/remix.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Connect a Remix application to Neon
+subtitle: Set up a Neon project in seconds and connect from a Remix application
+enableTableOfContents: true
+updatedOn: '2023-10-06T17:44:14.691Z'
+---
+
+Remix is a full stack web framework that lets you focus on the user interface and work back through web standards to deliver a fast, slick, and resilient user experience. People are gonna love using your stuff.
```

Comment:
Would you be open to making this a quote and linking to our website since this is the h1 copy?

Reply:
@danieltprice, that sounds like one for you.

Reply:
Makes sense to me. Let's try the quote and link. If it looks odd, we'll revise.